README.md

```diff
@@ -67,6 +67,7 @@ This API offers methods for creating, retrieving, listing and deleting resources
 |                | namespace |
 | list         | kind| returns a collection of resources of a given kind
 |                | namespace |
+| update         | spec object | updates an existing resource
```

Comment:
Should `apply` simply perform either a _create_ or _update_ depending upon existence? Have explicit calls to  `create` fail if a resource is already there.

Reply:
Apply recieves a yaml. Update receives an object. These are two different operations.